plugin.video.botchamania/addon.xml

```diff
@@ -27,7 +27,6 @@
     <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
```

Comment:
would be nice if you could adjust the license to a spdx compliant identifier at some point